src/custom/utils/explorer.ts

```diff
@@ -1,10 +1,10 @@
 import { SupportedChainId as ChainId } from 'constants/chains'
 import { OrderID } from 'utils/operator'
-import { isDev, isStaging } from './environments'
+import { isDev, isStaging, isPreStaging } from './environments'
 
 function _getExplorerUrlByEnvironment() {
   let baseUrl: string | undefined
-  if (isDev) {
+  if (isDev || isPreStaging) {
     baseUrl = process.env.REACT_APP_EXPLORER_URL_DEV || 'https://protocol-explorer.dev.gnosisdev.com'
   } else if (isStaging) {
```

Comment:
Shouldn't it use the same as staging?

Reply:
no, that's the whole point, they want to use their stable, they call it staging, points to their `main` branch, although their url has `dev` in it

From the website point of view, is `preStaging`, like a staging environment with the `dev API`

Reply:
You are right, my mistake, all good!